### PR TITLE
(DOCSP-29270): Enterprise Authentication page

### PR DESCRIPTION
**Pull request by cbullinger**
> NOTE: Like the code snippets from the corresponding Java page, 
> enterprise auth code snippets are untested 

# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-java/blob/master/REVIEWING.md)

JIRA - https://jira.mongodb.org/browse/DOCSP-29270
Staging - https://docs-mongodbcom-staging.corp.mongodb.com/kotlin/docsworker-xlarge/docsp-29270-enterprise-auth/fundamentals/enterprise-auth/

# Update Page
Fundamentals > [Enterprise Authentication Mechanisms page](https://www.mongodb.com/docs/drivers/java/sync/v4.9/fundamentals/enterprise-auth/)

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [ ] Did you run a spell-check?
- [ ] Did you run a grammar-check?
- [ ] Are all the links working?

### CODE REVIEW

File: source/fundamentals/enterprise-auth.txt [hunk 1]
```diff
@@ -84,9 +87,8 @@ mechanism:
 
       Your code to instantiate a ``MongoClient`` should look something like this:
```
Comment (rustagir) [inline source/fundamentals/enterprise-auth.txt:88]:
```suggestion
      Your code to instantiate a ``MongoClient`` should resemble the following:
```

File: source/fundamentals/enterprise-auth.txt [hunk 2]
```diff
@@ -267,15 +270,16 @@ mechanism:
```
Comment (rustagir) [inline source/fundamentals/enterprise-auth.txt:270]: cant comment there but on line 264 i think it looks better to italicize everything including the parentheses. Applies in another place as well

**Review by rustagir — CHANGES_REQUESTED** [review]
a few improvements

File: source/fundamentals/enterprise-auth.txt (hunk 1 above)
Comment (cbullinger) [reply source/fundamentals/enterprise-auth.txt:88]: Updated throughout

File: source/fundamentals/enterprise-auth.txt (hunk 2 above)
Comment (cbullinger) [reply source/fundamentals/enterprise-auth.txt:270]: Updated both

**Review by rozza — APPROVED** [review]
LGTM!

File: source/includes/fundamentals/auth-specify.rst [hunk 3]
```diff
@@ -24,7 +24,7 @@ connection settings when constructing your ``MongoClient``.  Select the
 For more information on these classes and methods, refer to the following API
```
Comment (rustagir) [inline source/includes/fundamentals/auth-specify.rst:24]: Optional suggestion: In the preceding paragraph, I think it's a little confusing to tell users to pick a tab when there aren't any tabs in this section -- maybe you can phrase it to tell users that in each section of the guide, there will be options to select either/or.

**Review by rustagir — APPROVED** [review]
LGTM! save one optional item

File: source/includes/fundamentals/auth-specify.rst (hunk 3 above)
Comment (cbullinger) [reply source/includes/fundamentals/auth-specify.rst:24]: Updated for clarity

**Pull request merged**